modules/multi-architecture-enabling-64k-pages.adoc

```diff
@@ -0,0 +1,84 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/multi-architecture-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="multi-architecture-enabling-64k-pages_{context}"]
+
+= Enabling 64k pages on the {op-system-first} kernel 
+
+You can enable 64k memory page in the {op-system-first} kernel on the 64-bit ARM compute machines in your cluster. The 64k page size kernel specification can be used for large GPU or high memory workloads. This is done using the Machine Config Operator (MCO) which uses a machine config pool to update the kernel To enable 64k page sizes, you must create a machine config pool for AMD64 and a machine config pool for ARM64.
+
+[IMPORTANT]
+====
+Using 64k pages is exclusive to 64-bit ARM architecture compute nodes or clusters installed on 64-bit ARM machines. If you configure the 64k pages kernel on a machine config pool using 64-bit x86 machines, the machine config pool and operator will degrade.
+====
+
+.Prerequisites: 
+* You installed the OpenShift CLI `oc`
+* You created a cluster with compute nodes of different architecture on one of the supported platforms
+
+.Procedure: 
+
+. Add a label to the node you want to assign as the infra node with a specific label: 
+//not sure if this part was needed
+[source,terminal]
++
+----
+$ oc label node <node_name> <label>
```

Comment:
should we specify the example label here?

```
node-role.kubernetes.io/worker-64k-pages: ""
```

Yes, this part is needed too.

I'm not sure whether to specify that in some cases (IPI clusters using the MachineApi), users can also define this label at MachineSet level.

Reply:
I think i understand your suggestion, lemme add that!